riocli/device/delete.py

```diff
@@ -122,7 +122,10 @@ def delete_device(
         spinner.write('')
 
         if failed_count == 0 and success_count == len(devices):
-            spinner_text = click.style('All devices deleted successfully.', Colors.GREEN)
+            if delete_all:
+                spinner_text = click.style('All devices deleted successfully.', Colors.GREEN)
+            else:
+                spinner_text = click.style('{} device(s) deleted successfully.'.format(len(devices)), Colors.GREEN)
```

Comment:
Let's just keep this instead.
```suggestion
            spinner_text = click.style('{} device(s) deleted successfully.'.format(len(devices)), Colors.GREEN)
```

nit: Please correct the scope for this commit
```
fix(device): improves message when devices deleted successfully
```